src/config.py

```diff
@@ -40,6 +40,8 @@ class Settings(BaseSettings):
     """
     upload_fs_root: str
 
+    lei_service_url: str = "http://sbl-project-user_fi-1:8888/v1/institutions/"
```

Comment:
let's add the key to `.env.template` as well, probably call it `institutions_api_url` or `user_fi_api_url`; since lei is an identifier, and the entity we're actually checking for is a `financial institution`

Reply:
Updated to user_fi_api_url and added to .env.template.